components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2934,6 +2943,25 @@ private void parseOAuthDeviceCodeGrantConfig(OMElement oauthElem) {
         }
     }
 
+    private void parseOAuthPARConfig(OMElement oauthElem) {
+
+        OMElement oauthPARElement = oauthElem
+                .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.PAR));
+
+        if (oauthPARElement != null && oauthPARElement
+                .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.PAR_EXPIRY_TIME)) != null) {
+            try {
+                parExpiryTime = Long.parseLong(oauthPARElement
+                        .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.PAR_EXPIRY_TIME)).getText()
+                        .trim());
+            } catch (NumberFormatException e) {
+                log.error("Error while converting par expiry " + oauthPARElement
+                        .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.PAR_EXPIRY_TIME)).getText()
+                        .trim() + " to long. Falling back to the default value.", e);
```

Comment:
where do we fall back to the default value?

Reply:
private long parExpiryTime = 60;